Topcoder/FoldingPaper2.cc

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <vector>
+#include <cmath>
+#include <utility>
+#define oo 1e8
+using namespace std;
+class FoldingPaper2 {
+	public:
+	vector <pair <int,int> > findDiv(int n) {
+		vector <pair <int, int> > div;
+		for (int i = 1; i <= sqrt(n); i++) {
+			if (n % i != 0)
+				continue;
+			div.push_back(make_pair(min(i , n/i), max(i , n/i)) );
+		}
+		return div;
+	}
+
+	int foldPaperSide (int side, int goal) {
+		if (goal > side)
+			return -1;
+		if (goal == side)
+			return 0;
+		if (goal > ceil(side/2 + side % 2) || goal == ceil(side/2 + side % 2) )
+			return 1;
+		
+		return 1 + foldPaperSide(ceil(side/2 + side % 2), goal);
+	}
+	int solve(int W, int H, int A) {
+		
+		//Find div of A
+		
```

Comment:
Lot of vertical space.
https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace